projects/fsastorefrontlib/src/cms-components/my-account/myaccount.module.ts

```diff
@@ -12,6 +12,7 @@ import { CmsPageGuard, PageLayoutComponent } from '@spartacus/storefront';
 import { UserRequestStoreModule } from '../../core/user-request/store/user-request-store.module';
 import { FSAddressBookModule } from './address-book';
 import { ClaimModule } from './claim/claim.module';
+import { DocumentModule } from './documents';
```

Comment:
Since we have document**S** folder, perhaps rename module to Documents?

Reply:
if we create one day document details page that wont work. I would not create reference between module and package naming